ir/src/main/scala/org/scalajs/ir/Definitions.scala

```diff
@@ -14,26 +14,73 @@ package org.scalajs.ir
 
 import scala.annotation.switch
 
+import Trees.isValidJSIdentifier
```

Comment:
Yes I was already considering renaming `Definitions` to `Names`, but perhaps we can do that later. Or if you prefer I can do it in this commit.

I wouldn't move `isValidJSIdentifier` now because it will soon become irrelevant for `Names`, and will only apply to `JSGlobalRef`s, as Names will allow any JVM name, which is virtually any Unicode string.

Reply:
Ok. Let's leave things as they are and re-asses once the name mangling changes are complete.